bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/accessories/HomekitWindowCoveringImpl.java

```diff
@@ -34,17 +37,23 @@
  * @author epike - Initial contribution
  */
 public class HomekitWindowCoveringImpl extends AbstractHomekitAccessoryImpl implements WindowCoveringAccessory {
+    private final boolean inverted;
```

Comment:
Does this need to be a field?

Reply:
good point. i used it in the first version in some "IF"s but then i found a solution without "IF". so, dont need anymore. removing